src/js/items/Component.js

```diff
@@ -31,7 +31,10 @@ lm.utils.copy( lm.items.Component.prototype, {
 	},
 
 	setSize: function() {
-		this.container._$setSize( this.element.width(), this.element.height() );
+		if( this.element.is( ':visible' ) ) {
+			// Do not update size of hidden components to prevent unwanted reflows
+			this.container._$setSize( this.element.width(), this.element.height() );
```

Comment:
From what I see in my app, the hidden tab contents are re-sized, once they get visible.